packages/auto-approve/test/check-pr.test.ts

```diff
@@ -181,16 +199,73 @@ describe('check pr against config', async () => {
           resolve(fixturesPath, 'config', 'valid-schemas', 'valid-schema2.yml'),
           'utf8'
         )
-      ) as {rules: ValidPr[]};
+      ) as {rules: checkPR.ValidPr[]};
 
       const pr = require(resolve(
         fixturesPath,
         'events',
         'pull_request_opened_right_author_and_title_partial_schema'
       ));
 
-      const prMatchesConfig = await checkPRAgainstConfig(validPR, pr, octokit);
+      const scopes = listChangedFilesPR(200, [
+        {filename: 'README.md', sha: '1234'},
+        {filename: '.github/readme/synth.metadata/synth.metadata', sha: '1234'},
+      ]);
+
+      const prMatchesConfig = await checkPR.checkPRAgainstConfig(
+        validPR,
+        pr,
+        octokit
+      );
+      scopes.done();
+      assert.ok(prMatchesConfig);
+    });
+  });
+
+  describe('additional validation checks', async () => {
+    it('should correctly identify a PR that needs special attention', async () => {
```

Comment:
See my above comment, I think we could delve deeper into the algorithm in this set of assertions 👍